components/templates/src/global_fns/load_data.rs

```diff
@@ -93,11 +94,12 @@ impl DataSource {
     fn from_args(
         path_arg: Option<String>,
         url_arg: Option<String>,
+        literal_arg: Option<String>,
         base_path: &Path,
         theme: &Option<String>,
         output_path: &Path,
     ) -> Result<Option<Self>> {
-        if path_arg.is_some() && url_arg.is_some() {
+        if !(path_arg.is_some() ^ url_arg.is_some() ^ literal_arg.is_some()) {
```

Comment:
let's keep using `&&`

Reply:
Ok refactored representing "Exactly one of `path`, `url`, or `literal`" from using XOR to using AND's + OR's.